src/mountfsd/mountwork.c

```diff
@@ -489,6 +489,15 @@ static int vl_method_mount_image(
                                 /* image_filter= */ NULL,
                                 dissect_flags,
                                 &di);
+                if (r == -ENOPKG && !(dissect_flags & DISSECT_IMAGE_NO_PARTITION_TABLE)) /* Maybe it's a bare filesystem, try again */
```

Comment:
i don't get it? you open this up for anyone, which is already broken (this should never be allowed for unpriv users, since this can never find verity info. moreover, if you allow this anyway if you git ENOPKG, why not just add DISSECT_IMAGE_NO_PARTITION_TABLE to the flags in the first place right-away?

Reply:
I can update the commit message, but this is already restricted to root callers only by the image policy, given passing just an erofs is an 'unprotected' image, and the existing policy handling rejects it for non root

Reply:
I could also add a negative test to ensure that's the case if you like?